hugr-core/src/ops/custom.rs

```diff
@@ -108,24 +111,35 @@ impl ExtensionOp {
             name: self.def.name().clone(),
             description: self.def.description().into(),
             args: self.args.clone(),
-            signature: self.signature.clone(),
+            signature: self.signature.func_type.clone(),
+            static_inputs: self.signature.static_inputs.clone(),
         }
     }
+
+    /// Returns the [`ExtOpSignature`] of this [`ExtensionOp`].
```

Comment:
nit: We can already see the types.
```suggestion
    /// Return the instantiated signature of this [`ExtensionOp`].
```